app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Apparently you can just use `doctype strict`.

http://www.rubydoc.info/gems/slim/frames#__DOCTYPE__declaration

Do you know what the default is or why its not what we want?